app/src/main/java/org/mozilla/focus/widget/SwipeDownListener.kt

```diff
@@ -10,6 +10,10 @@ import kotlin.math.abs
 
 /**
  * Executes [doOnSwipeDown] when down swipes are detected.
+ *
+ * TODO this was written to be used in #106, but related functionality was
```

Comment:
I kind of want to remove this: I don't love confusing, unused code, especially if it accidentally sticks around for a few months, years. Maybe remove and squish those commits.

We can add it in another branch if you want to keep it around: maybe add it as a WIP comment in the other bug.